server/cli/src/helpers.rs

```diff
@@ -0,0 +1,23 @@
+use std::process::{Command, ExitStatus, Stdio};
+
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum CommandError {
+    #[error(transparent)]
```

Comment:
this one was moved from call_plugin to be shared with report cli to actually test for failed status